app/configuration/UUIDSessionCustomizer.java

```diff
@@ -18,18 +18,18 @@
  * Caused by: org.postgresql.util.PSQLException: ERROR: column "hierarchy_entry_id" is of type uuid but expression is of type character varying
  * Hint: You will need to rewrite or cast the expression.
  */
-public class UUIDSessionCustomizer implements SessionCustomizer {
-
-    @Override
-    public void customize(Session session) throws Exception {
-        ClassDescriptor classDescriptor = session.getClassDescriptor(DimensionValue.class);
-        DatabaseMapping mapping = classDescriptor.getMappingForAttributeName("hierarchyEntry");
-        if (mapping instanceof ManyToOneMapping) {
-            Vector<DatabaseField> foreignKeyFields = ((ManyToOneMapping) mapping).getForeignKeyFields();
-            if (!foreignKeyFields.isEmpty()) {
-                DatabaseField field = foreignKeyFields.elementAt(0);
-                field.setSqlType(Types.OTHER);
-            }
-        }
-    }
-}
+//public class UUIDSessionCustomizer implements SessionCustomizer {
+//
+//    @Override
+//    public void customize(Session session) throws Exception {
+//        ClassDescriptor classDescriptor = session.getClassDescriptor(DimensionValue.class);
+//        DatabaseMapping mapping = classDescriptor.getMappingForAttributeName("hierarchyEntry");
+//        if (mapping instanceof ManyToOneMapping) {
+//            Vector<DatabaseField> foreignKeyFields = ((ManyToOneMapping) mapping).getForeignKeyFields();
+//            if (!foreignKeyFields.isEmpty()) {
+//                DatabaseField field = foreignKeyFields.elementAt(0);
+//                field.setSqlType(Types.OTHER);
+//            }
+//        }
+//    }
+//}
```

Comment:
I've removed this file completely now as it is EclipseLink-specific. Can be resurrected from git history if we ever decide to go back to EclipseLink.